sdk/triggers.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { getTriggersListUrl, getSpecificTriggerUrl } = require('./url');
+const { getValidatedBody } = require('./handleError');
+
+async function upsert(apiKey, triggerName, triggerOptions) {
+  const listOptions = {
+    url: getSpecificTriggerUrl(triggerName),
```

Comment:
I would name it `getTriggerSpecificUrl` but it's not a big deal

Reply:
Ack